### PR TITLE
2022-02-26-sbiobertresolve_atc_en_2_4

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps drugs entities to ATC (Anatomic Therapeutic Chemical) codes using sbiobert_base_cased_mli Sentence Bert Embeddings.
+
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md:21]: `sbiobert_base_cased_mli`

File: docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+val posology_ner = MedicalNerModel.pretrained("ner_posology", "en", "clinical/models")
+      .setInputCols(Array("sentence", "token", "word_embeddings"))
+      .setOutputCol("ner")
+
+val ner_converter = NerConverterInternal()
+      .setInputCols(Array("sentence", "token", "ner"))
+      .setOutputCol("ner_chunk")
+      .setWhiteList(Array('DRUG'))
+
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md:124]: use "

File: docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+val c2doc = Chunk2Doc()
+      .setInputCols("ner_chunk")
+      .setOutputCol("ner_chunk_doc") 
+
+val sbert_embedder = BertSentenceEmbeddings.pretrained('sbiobert_base_cased_mli', 'en','clinical/models')
+      .setInputCols("ner_chunk_doc")
```
Comment (muhammetsnts) [inline docs/_posts/Cabir40/2022-02-26-sbiobertresolve_atc_en_2_4.md:130]: use "

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

**Review by muhammetsnts — APPROVED** [review]
Good job!

**Review by galiph — APPROVED** [review]
Thanks

**Comment by muhammetsnts** [comment]
@galiph will train the augmented version and we will upload that model.

**Pull request closed without merging**